src/main.rs

```diff
@@ -18,10 +14,17 @@ fn run() -> Result<()> {
         .map(String::from);
 
     match process_name.as_deref() {
-        Some(component::FUELUP) => fuelup_cli::fuelup_cli()?,
+        Some(component::FUELUP) => {
+            if let Err(e) = fuelup_cli::fuelup_cli() {
+                bail!("{}", e);
+            }
+        }
```

Comment:
Out of curiosity, why was this branch changed? The original

```rust
Some(component::FUELUP) => fuelup_cli::fuelup_cli()?,
```
appears to behave the same no?

Reply:
This was an error, it was meant to be an `error!` instead of a `bail!` (to show errors to the user)

Reply:
At first I confused myself and removed this `error!` change again ([8fe3741](https://github.com/FuelLabs/fuelup/pull/73/commits/8fe37418ee73f78751e3af3a2f015762355c31a4)). While working on `fuelup check` i realized the reason why I included the `error!` was to display the error (which would otherwise be concealed) so I'm adding it back again